config/nav.yml

```diff
@@ -154,7 +153,6 @@ nav:
         - GitHub Webhook - Go: serving/samples/gitwebhook-go/README.md
         - gRPC Server - Go: serving/samples/grpc-ping-go/README.md
         - Hello world apps:
-          - Overview: serving/samples/hello-world/README.md
```

Comment:
just to triple check, you intended to remove these two overviews in this PR? (just because it seems different from the rest of the changes; rest lgtm)

Reply:
Yeah, those readmes only had YAML frontmatter and nothing else in them, so there were two I removed like this IIRC

Reply:
aha, cool, lgtm then!